packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -172,6 +172,10 @@ export class Span implements types.Span, ReadableSpan {
     return true;
   }
 
+  isEnded(): boolean {
```

Comment:
Where are we using this method?

Reply:
In my SpanProcessor/Exporter (which is not part of this repo) which exports additionally spans which are not yet finished to allow backend to correlate/show also long running spans before they are finished.

Reply:
why don't we use `get ended() { ... }` method instead.

Reply:
good catch! I forgot to remove it during removing the API changes. It was there to have something symetric to `isRecording`. It's removed now.